Dockerfile

```diff
@@ -1,20 +1,23 @@
-ARG NODE_IMAGE=node:12.2-alpine
+ARG NODE_IMAGE=node:10.16-alpine
 ARG WORKDIR=/usr/src/app/
+ARG FLOOD_BASE_URI=/
 
 FROM ${NODE_IMAGE} as nodebuild
 ARG WORKDIR
+ENV FLOOD_BASE_URI $FLOOD_BASE_URI
 
 WORKDIR $WORKDIR
 
 # Generate node_modules
 COPY package.json \
-     package-lock.json \
-     .babelrc \
-     .eslintrc.js \
-     .eslintignore \
-     .prettierrc \
-     ABOUT.md \
-     $WORKDIR
+    package-lock.json \
```

Comment:
nit: Can we restore the previous, unless there's a good reason to change it?

Reply:
The previous indentation has 5 spaces. The rest of file has 4 spaces, i think it should use the same. What do you think? If you want to mantain 5 spaces in these lines, i can revert it.